frontend/test/__support__/server-mocks/collection.ts

```diff
@@ -85,7 +85,7 @@ export function setupCollectionByIdEndpoint({
     return;
   }
 
-  fetchMock.get(/api\/collection\/\d+/, url => {
+  fetchMock.get(/api\/collection\/\d+|root/, url => {
     const collectionIdParam = url.split("/")[5];
     const collectionId = Number(collectionIdParam);
```

Comment:
Do we need to special case the root collection here? Notice how it's coercing the `collectionIdParam` to a number, and then uses it to find a collection object below. For root collection, it seems like we'll end up with something like that:

```js
const collectionIdParam = "root"
const collectionId = Number(collectionIdParam) // NaN
collections.find(c => c.id === collectionId) // always undefined
```